build-system/tasks/sweep-experiments/index.js

```diff
@@ -241,8 +241,8 @@ const findConfigBitCommits = (
       tokens.pop();
     }
     return {
-      hash: tokens.shift(),
-      authorDate: tokens.shift(),
+      hash: /** @type {string} */ (tokens.shift()),
```

Comment:
Unlike my the other comment I left about string.split.shift this could technically be undefined but I trust that the @alanorozco understands this code better than I.

Reply:
Sorry, I had this missed this question!

This seems fine per format received: `%h %aI %s`, though we have the conditional `.pop()` above. I'd say this is good but I'll make a separate change to put up a guard before this cast.

Reply:
#33266 for safer parsing/testing

Reply:
Thanks @alanorozco